src/variable.cpp

```diff
@@ -188,8 +193,6 @@ void Variable::VariableImpl::unsetEquivalentTo(const VariablePtr &equivalentVari
     auto result = findEquivalentVariable(equivalentVariable);
     if (result != mEquivalentVariables.end()) {
         mEquivalentVariables.erase(result);
-    } else {
-        throw std::out_of_range("Equivalent variable not found.");
```

Comment:
The fact that we used to throw an exception tells me that we should now return a boolean status. For good measure, we should probably do the same for `setEquivalentTo()`. Then, we ought to modify accordingly anything that in/directly uses `un/setEquivalentTo()`.